packages/wdio-utils/src/driver/manager.ts

```diff
@@ -47,15 +55,19 @@ function mapCapabilities (
         /**
          * only set up driver if
          */
+        // - capabilities are defined and not empty because automationProtocol is set to `devtools`
+        cap &&
         // - browserName is defined so we know it is a browser session
         cap.browserName &&
         // - we are not about to run a cloud session
         !definesRemoteDriver(options) &&
         // - we are not running Safari (driver already installed on macOS)
         !isSafari(cap.browserName) &&
         // - driver options don't define a binary path
-        !getDriverOptions(cap).binary
-    ))
+        !getDriverOptions(cap).binary &&
+        // - user is not defining "devtools" as automation protocol
+        (options as Options.WebdriverIO).automationProtocol !== 'devtools'
```

Comment:
Should the `setup*()` methods and this one receive an `options: Options.WebdriverIO` so you don't have to do this casting?

Reply:
in hindsight, yes .. good catch, will push a fix.